kong/tools/request_aware_table.lua

```diff
@@ -33,7 +33,7 @@ local function enforce_sequential_access(table)
     return
   end
 
-  local curr_request_id = var.request_id
+  local curr_request_id = var.kong_request_id
```

Comment:
Perhaps `tracing.request_id.get()` is a better choice? @samugi.

Reply:
+1 for `tracing.request_id.get()`, but don't forget to cache this function.

Reply:
done

Reply:
yes that's probably better @chronolaw